addons/popochiu/engine/interfaces/i_room.gd

```diff
@@ -94,7 +101,7 @@ func get_runtime_room(script_name: String) -> PopochiuRoom:
 	
 	var rp: String = PopochiuResources.get_data_value('rooms', script_name, null)
 	if rp.is_empty():
-		printerr('[Popochiu] No PopochiuRoom with name: %s' % script_name)
+		PopochiuUtils.print_error('No PopochiuRoom with name: %s' % script_name)
```

Comment:
I would add some context to this error message. "No PopochiuRoom with name %s"... while doing what? What happens now?

Reply:
Done... I hope so.